### DISCUSSION TITLE
[deepin-terminal] open folder and edit file from terminal directly

### DISCUSSION THREAD

**Discussion by aohanhongzhi** [linuxdeepin/developer-center#7013] [Features Request & Ideas | 特性请求 & 头脑风暴]
FYI  https://docs.warp.dev/getting-started/readme

<img width="506" alt="image" src="https://github.com/linuxdeepin/developer-center/assets/37319319/4df68257-12fd-46ba-88c2-ec6fcdab5332">

<img width="248" alt="image" src="https://github.com/linuxdeepin/developer-center/assets/37319319/ccb0081d-c9ff-4ab7-8b89-93624f1982de">

**Comment by BLumia** [comment]
I assume this is a suggestion apply to deepin-terminal project, so I edited the discussion title. Feel free to edit if it's not related to deepin-terminal.

cc @ArchieMeng
